sesman/config.c

```diff
@@ -100,11 +106,12 @@ config_read(struct config_sesman *cfg)
 
 /******************************************************************************/
 int DEFAULT_CC
-config_read_globals(int file, struct config_sesman *cf, struct list *param_n,
+config_read_globals(char *base_dir, int file, struct config_sesman *cf, struct list *param_n,
```

Comment:
Please always use `const char *` for string arguments unless the string is used for output.